src/test/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/GOAGeneratorUtilitiesTest.java

```diff
@@ -14,7 +14,10 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.*;
+import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorConstants.C_TRACHOMATIS_CROSS_REFERENCE;
 
 @RunWith(PowerMockRunner.class)
```

Comment:
I don't think this test class uses PowerMockito, so this `@RunsWith` is probably not necessary. Also, see my earlier comment about removing PowerMockito.

Reply:
Okay, I replaced this with `@RunWith(MockitoJUnitRunner.class)` and the tests still are okay.

Reply:
I think you might be able to remove the `@RunsWith` entirely.

Reply:
It is needed since removal leads to the tests failing.  I think this inits the mocks, so it can be done with the annotation or in code in a `@Before` block.

Reply:
Ok. There are probably ways to get rid of it, but it's not a priority.